lib/pacemaker/pcmk_sched_clone.c

```diff
@@ -39,39 +39,45 @@ add_dependent_scores(gpointer data, gpointer user_data)
 
 /*!
  * \internal
- * \brief Assign a clone resource to a node
+ * \brief Assign a clone resource's instances to nodes
  *
- * \param[in,out] rsc     Resource to assign to a node
+ * \param[in,out] rsc     Clone resource to assign
  * \param[in]     prefer  Node to prefer, if all else is equal
  *
- * \return Node that \p rsc is assigned to, if assigned entirely to one node
+ * \return NULL (clones are not assigned to a single node)
  */
 pe_node_t *
-pcmk__clone_allocate(pe_resource_t *rsc, const pe_node_t *prefer)
+pcmk__clone_assign(pe_resource_t *rsc, const pe_node_t *prefer)
 {
+    GList *iter = NULL;
```

Comment:
We only use this once if you wanted to keep it in the loop header. Not worth bothering with unless you're already changing something else.

Reply:
thanks